pkg/kdump/kdump-client.js

```diff
@@ -106,19 +100,24 @@ export class KdumpClient {
             path = "/var/crash";
 
         return new Promise((resolve, reject) => {
-            if (target.target === "local") {
+            if (target.type === "local") {
                 // local path, try to see if we can write
                 cockpit.script(testWritableScript, [path], { superuser: "try" })
                         .then(resolve)
                         .catch(() => reject(cockpit.format(_("Directory $0 isn't writable or doesn't exist."), path)));
                 return;
-            } else if (target.target === "nfs") {
-                if (!target.nfs.value.match("\\S+:/.+"))
-                    reject(_("nfs dump target isn't formatted as server:path"));
-            } else if (target.target === "ssh") {
-                if (!target.ssh.value.trim())
+            } else if (target.type === "nfs") {
+                if (!target.server || !target.server.trim())
+                    reject(_("nfs server is empty"));
+                // IPv6 must be enclosed in square brackets
+                if (target.server.trim().match(/^\[.*[^\]]$/))
+                    reject(_("nfs server is not valid IPv6"));
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17544-20220719-143653-7a0d5780-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)